spec/polaris-management-service.yml

```diff
@@ -868,6 +868,8 @@ components:
           description: URI to the remote catalog service
         authenticationParameters:
           $ref: "#/components/schemas/AuthenticationParameters"
+        serviceIdentity:
```

Comment:
Shouldn't this field be marked as read-only?

Reply:
The ref schema is read-only already, so we should be good.